converter/src/main/java/gov/cms/qpp/conversion/model/error/Detail.java

```diff
@@ -1,16 +1,23 @@
 package gov.cms.qpp.conversion.model.error;
 
+import java.io.Serializable;
+import java.util.Objects;
+
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
 
-import java.io.Serializable;
+import gov.cms.qpp.conversion.model.Node;
 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'gov.cms.qpp.conversion.model.TemplateId'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)